test/cookbooks/python-webapp-test/recipes/default.rb

```diff
@@ -15,3 +15,51 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+include_recipe 'yum-ius'
+include_recipe 'yum-epel'
+
+%w(gcc python27 python27-devel python27-pip).each do |pkg|
+  package pkg
+end
+
+group 'chef' do
+  action :create
+end
+
+user 'chef' do
+  action :create
+  gid 'chef'
+end
+
+path = '/opt/tutorial-a'
+
+hash = {
+  'path' => path,
+  'engine' => 'django.db.backends.sqlite3',
+  'dbname' => "#{path}/yourdatabasename.db"
+}
+
+python_webapp 'tutorial-a' do
+  create_user true
+  owner 'tutorial-a'
+  group 'tutorial-a'
+
+  repository 'https://github.com/osuosl/python-test-apps.git'
+
+  config_template 'config.yml.erb'
+  config_destination "#{path}/config.yml"
+  config_vars hash
+  django_migrate true
+  django_collectstatic true
+  interpreter 'python2.7'
+  revision 'cookbook_test'
+end
+
+python_webapp 'tutorial-b' do
+  path '/opt/tutorial-b'
```

Comment:
You should set this to a path that isn't just /opt/. that's why we have this one :)